ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -704,7 +700,7 @@ public void shouldFailForInvalidTerminateClusterParameters() {
     final KsqlRequest ksqlRequest = new KsqlRequest(TerminateCluster.TERMINATE_CLUSTER_STATEMENT_TEXT, properties);
     final KsqlResource testResource = TestKsqlResourceUtil.get(ksqlConfig, ksqlEngine);
     final Response response = testResource.terminateCluster(ksqlRequest);
-    assertThat(response.getEntity().toString(), equalTo("You need to send both SOURCES_LIST and SOURCES_LIST_TYPE parameters as part of your request.\n"));
+    assertThat(response.getStatus(), equalTo(200));
```

Comment:
name of test suggests negative test, but you're checking for 200 status, which is success.